browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -4774,7 +4762,7 @@ L.CanvasTileLayer = L.Layer.extend({
 	},
 
 	_addDropDownMarker: function () {
-		if (this._validatedCellXY && this._cellCursorXY && this._validatedCellXY.equals(this._cellCursorXY)) {
+		if (this._validatedCellAddress && app.file.calc.cellCursor.visible && this._validatedCellAddress.equals(app.file.calc.cellCursor.address.toArray())) {
```

Comment:
SimpleRectangle has method: `equals`, can we avoid `toArray` this way?

Reply:
(seen few times in this PR)

Reply:
If i do this, i will probably need to convert L.Point to SimplePoint in multiple places (in place or change the variable, doesn't matter). I am not sure how frequently it will happen. That's why i am trying to push this to future adjustments :D Not that i don't agree.

I am currently working on a little more complicated part of cell cursor. Let me re-think while i work on the next PR.

I need/want to feel more comfortable with variable interoperability before i enforce things.